shared/settings/feedback-container.native.js

```diff
@@ -15,7 +15,7 @@ import {
   logFileName,
   traceDir,
 } from '../constants/platform'
-import {type TimerProps} from '../common-adapters/hoc-timers'
+import {type TimerProps, default as HOCTimers} from '../common-adapters/hoc-timers'
```

Comment:
same here

Reply:
Done, also searched-and-replaced other instances of this pattern